docs/building-from-source.md

```diff
@@ -146,9 +146,7 @@ $ build.py --host-toolchain mingw --copy-runtime-dlls no
 
 ## Divergences from upstream
 
-### newlib:
-* Clang does not support the ``naked`` attribute on C functions, breaking the
-  Linux startup (out of scope).
+### picolibc:
 * Target triple ending with eabi is not considered an ELF target.
```

Comment:
I have no idea.

I got rid of the "breaking Linux startup" bullet because it looks like picolibc doesn't have such startup. But I don't understand the "Target triple ending with eabi is not considered an ELF target" bullet so I can't judge whether it still applies.

Reply:
Removed. Now only the patch to enable building with -mthumb is described here.